### ISSUE TITLE
endless loop - can't reset acccount/passphrase

### ISSUE THREAD

**Issue by aaomoware** [keybase/keybase-issues#2707]
Hi,

I have forgotten my passphrase and I am not actively logged into any of my devices. 

1. When i try to reset my account I am asked for the same passphrase i can't remember - https://keybase.io/#account-reset

2. Clicked on the https://keybase.io/#no forgot Passphrase; two options are presented in other to recover a lost/forgotten passphrase
  a. keybase passphrase recover  - which also asks, on its third prompt, the same passphrase i had forgotten
  b. Email or username   - which doesn't work as it complains of an active device.

"You can't reset your passphrase, since you have active device or paper keys. The following devices all have the power to change your passphrase.
UbuntuDesktop
If you have lost all these devices, please contact max@keybase.io"

So in a nutshell, aside the help of keybase from behind the scene, this appears to be an endless loop troubleshooting.

**Comment by scottstensland** [comment]
does anyone ever get a response back from max@keybase.io  ?    I too have above issue ... keybase is setup on my desktop computer however I am away traveling so need a way to reset password via email

**Comment by maxtaco** [comment]
@scottstensland I believe I replied to you this am, but if not, email me
